build/rollup_plugins.js

```diff
@@ -29,6 +29,7 @@ export const plugins = () => [
         include: 'src/shaders/index.js'
     }),
     commonjs({
+        ignoreGlobal: true,
```

Comment:
Add a comment here outlining why we're using this option and linking to this PR.